README.rst

```diff
@@ -1,20 +1,254 @@
+            widget: {
+                id: 'custom_footer',
+                type: DIRECT_PLUGIN,
+                RenderWidget: SlotWidgetFooter,
+            },
+        }
+        """,
+    }
+
+    default = learner_dashboard_slots
+
+These definitions allow inserting custom widgets like `SlotWidgetLearnerDashboardSidebar` and `SlotWidgetFooter`
+into MFEs without modifying core Open edX code. The plugin dynamically loads these slots during initialization to
+ensure a seamless integration with the frontend experience.
+
+Slots availables:
+^^^^^^^^^^^^^^^^^
+
+The slots available for each MFE can be found in the master branch, within the `src/plugin-slots` directory.
+For example, https://github.com/openedx/frontend-app-learner-dashboard/tree/master/src/plugin-slots
+
+For MFEs that function as shared components, such as the Header and Footer, the slots defined in them can
+be used directly by the parent MFE that integrates them.
+
+For example, the `logo_slot` slot is available within the frontend-component-header and can be used in the
+Learning MFE, provided that the MFE has a version of the frontend-component-header that includes this slot.
+This allows greater flexibility in customizing MFEs without directly modifying their source code.
+
+Patches
+-------
+
+This plugin applies several patches to ensure proper theme integration. Here are some key patches and their functions:
+
+- openedx-lms-production-settings:
+
+.. code-block:: json
+
+    ENABLE_COMPREHENSIVE_THEMING = True
+    COMPREHENSIVE_THEME_DIRS.extend("/openedx/themes/modern-theming")
+
+This enables comprehensive theming and ensures the modern-theming directory is included in the theme search path.
+
+- mfe-dockerfile-post-npm-install:
+
+.. code-block:: json
+
+    RUN git clone https://github.com/eduNEXT/frontend-render-widgets.git
+    RUN npm install ./frontend-render-widgets
+
+This ensures that the frontend-render-widgets repository is cloned and installed, providing the necessary JSX components for slot rendering.
+
+- mfe-env-config-runtime-definitions:
+
+.. code-block:: js
+
+    const { SlotWidgetHeaderLogo, SlotWidgetFooter, SlotWidgetLearnerDashboardSidebar } = require('./frontend-render-widgets/src');
+
+This imports custom JSX components from frontend-render-widgets, making them available for use within MFEs.
+
+- mfe-lms-production-settings & mfe-lms-development-settings:
+
+.. code-block:: json
+
+    MFE_CONFIG["PARAGON_THEME_URLS"] = {
+        "core": {
+            "url": "https://cdn.jsdelivr.net/npm/@edunext/modern-theming-alpha@1.0.0/dist/core.min.css"
+        },
+        "defaults": {
+            "light": "light"
+        },
+        "variants": {
+            "light": {
+                "url": "https://cdn.jsdelivr.net/npm/@edunext/modern-theming-alpha@1.0.0/dist/light.min.css"
+            }
+        }
+    }
+
+These patches configure Paragon-based theming for both production and development environments.
+
+Customization
```

Comment:
Can we have an example on how to customize our theming with this plugin? I just see https://github.com/ZachTechGuru/tribuga-namespace/pull/29/files where we install and enable the plugin, can we put an example of some customization ?

Reply:
Do you mean screenshots or how to use the plugin?

Reply:
yes, just a little plugin  changing something